lib/protobuf/enum.rb

```diff
@@ -120,7 +127,8 @@ def self.enum_for_name(name)
     #   Enums, the first enum defined will be returned.
     #
     def self.enum_for_tag(tag)
-      enums_for_tag(tag).first
+      value = mapped_enums[tag.to_i]
```

Comment:
`nil.to_i` is zero ... I don't think we want that here. can we guard on `if tag` ?